app-config.yaml

```diff
@@ -53,6 +53,13 @@ newrelic:
 lighthouse:
   baseUrl: http://localhost:3003
 
+scaffolder:
```

Comment:
also in https://github.com/spotify/backstage/pull/2492, let's make sure it doesn't get merged in twice